src/Rules.hs

```diff
@@ -29,6 +30,7 @@ allStages = [minBound ..]
 -- or Stage1Only targets
 topLevelTargets :: Rules ()
 topLevelTargets = action $ do
+    need inplaceLibCopyTargets
```

Comment:
`need` is a blocking operation, so it's better to put all targets together and do a single top-level `need` -- see the code below.